index.bs

```diff
@@ -5114,35 +5114,40 @@ a [=user-verifying platform authenticator=].
 
 1. The [=[RP]=] script runs the code snippet below.
 
-1. The user agent asks the user whether they are willing to register with the [=[RP]=] using an available [=platform authenticator=].
+    1. The user agent checks if a [=user-verifying platform authenticator=] is available.
 
-1. If the user is not willing, terminate this flow.
+    1. The [=[RP]=] asks the user if they want to create a credential with it.
```

Comment:
Hm, do we really need this check here -- the "appropriate UI ... guides user" is materialized down below in line 5124, yes?  Also, the thrust of issue #1178 is that we are not prompting the user, yes?

Reply:
Yup, we are not prompting the user.

Reply:
Note that the subject here is the RP, not the client - this would happen before the RP makes the credentials.create() call. For example, by showing a nudge banner, interstitial page or similar.

Reply:
Ah.. ok.

Reply:
Ok, understood, scratch this particular review.